v3.0/benchmark/how-to-run-tpcc.md

```diff
@@ -31,18 +31,24 @@ TPC-C uses tpmC (transactions per minute) to measure the maximum qualified throu
 
 This testing uses the open-source BenchmarkSQL 5.0 as the TPC-C testing tool and adds the support for the MySQL protocol. You can download the testing program by using the following command:
 
+{{< copyable "shell-regular" >}}
+
 ```shell
 git clone -b 5.0-mysql-support-opt-2.1 https://github.com/pingcap/benchmarksql.git
 ```
 
 To install Java and Ant, run the following command (take CentOS as an example):
 
+{{< copyable "shell-regular" >}}
+
 ```shell
 sudo yum install -y java ant
 ```
 
 Enter the `benchmarksql` directory and run `ant` to build:
 
+{{< copyable "shell-regular" >}}
+
 ```shell
 cd benchmarksql
```

Comment:
```suggestion
cd benchmarksql &&
```
NOTE that you need to add `&&` in between if the block has multiple lines of commands.